astro/src/content/docs/operate/deploy/_theme-environment-management.mdx

```diff
@@ -0,0 +1,55 @@
+When moving themes from one environment to another, the theme logic and look and feel may be the same, but the assets may be different. FusionAuth does not resolve environment variables in the themes, or anything other than [what is documented](/docs/customize/look-and-feel/template-variables).
```

Comment:
```suggestion
When moving themes from one environment to another, the theme logic and look and feel may be the same, but the assets may be different. FusionAuth resolves environment variables in the themes that are [documented](/docs/customize/look-and-feel/template-variables).
```
Try to turn this into what FusionAuth does vs what it doesn't do.

Reply:
Thanks, made it clearer.